testing/citest/tests/kube_smoke_test.py

```diff
@@ -527,6 +690,22 @@ def test_c_create_server_group(self):
   def test_d_create_find_image_pipeline(self):
     self.run_test_case(self.scenario.create_find_image_pipeline())
 
+  def test_cd_save__statefulsets_pipeline(self):
+    self.run_test_case(self.scenario.save_statefulset_pipeline())
+
+
+  def test_d_create_statefulsets_pipeline(self):
+    self.run_test_case(self.scenario.create_statefulset_pipeline())
+
+
+  def test_cd_save__daemonset_pipeline(self):
```

Comment:
This is going to execute before the statefulsets because of the lexical ordering.
That's confusing so I'd rename and/or reorder the tests so that they are in the order you intend to execute as well as lexographic.

The long term intention is to specify the dependencies of the tests and the runner will perform the partial ordering to satisfy the constraints. However I never got around to implementing that and the tests have generally been small and simple enough that the lexographic ordering was good enough.